### ISSUE TITLE
Topojson Croatia administrative borders

### ISSUE THREAD

**Issue by schlos** [openzagreb/proracun#1]
We need help in getting right structure in hrv.topo,json like seen in deu.topo.json with highest administrative level borders and names (zupanije).

It should be accomplished using topojson, like described on this guide (UK): http://bost.ocks.org/mike/map/ and this guide (DE): http://milkator.wordpress.com/2013/02/25/making-a-map-of-germany-with-topojson/

hrv.topo.json will be used to generate dynamic svg with Croatian administrative borders.

**Comment by Owlree** [comment]
Is this one still needed?

**Comment by schlos** [comment]
Yes, it's still needed. We're running demo site without this map as I was unable to properly build topojson.

This is our demo site: http://proracun.org/index.html
With correct topojson file, python will build static html with SVG out of topojson like seen on German budget page: https://offenerhaushalt.de/

Let me know if you need any additional information.

Repo branch is "croatia".
